CIP-1694/README.md

```diff
@@ -0,0 +1,1142 @@
+
+<!-- TODO: Provide a CBOR specification in the annexe for these new on-chain entities -->
+
+In addition, each action will include some elements that are specific to its type:
+
+| Governance action type         | Additional data                                               |
+| :--                            | :--                                                           |
+| 1. Motion of no-confidence     | None                                                          |
+| 2. New committee/quorum        | The set of verification key hash digests and a quorum number  |
+| 3. Update to the Constitution  | A hash digest of the Constitution document                    |
+| 4. Hard-fork initiation        | The new (greater) major protocol version                      |
+| 5. Protocol parameters changes | The changed parameters                                        |
+| 6. Treasury withdrawal         | A map from stake credentials to a positive number of Lovelace |
+| 7. Info                        |                                                               |
+
+Each accepted governance action will be assigned a unique identifier (a.k.a. the **governance action ID**),
+consisting of the transaction hash that created it and the index within the transaction body that points to it.
+
+> ** Note: There can be no duplicate committee members - each pair of credentials in a committee must be unique.
+
+#### Protocol Parameter groups
+
+We have grouped the protocol parameter changes by type.
+This allows different thresholds to be set for each group, and also supports separate votes.
+DReps may choose to abstain to vote on parameter changes that are outside of their field of expertise.
+The _network_,  _economic_ and _technical_ parameter groups collect existing protocol parameters that were introduced during the Shelley, Alonzo and Babbage eras.
+In addition, we introduce a new _governance_ group that is specific to the new governance parameters that will be introduced by CIP-1694.
+
+The **network group** consists of:
+* maximum block body size (`maxBBSize`)
+* maximum transaction size (`maxTxSize`)
+* maximum block header size (`maxBHSize`)
+* maximum size of a serialized asset value (`maxValSize`)
+* maximum script execution units in a single transaction (`maxTxExUnits`)
+* maximum script execution units in a single block (`maxBlockExUnits`)
+
+The **economic group** consists of:
+* minimum fee coefficient (`minFeeA`)
+* minimum fee constant (`minFeeB`)
+* delegation key Lovelace deposit (`keyDeposit`)
+* pool registration Lovelace deposit (`poolDeposit`)
+* monetary expansion (`rho`)
+* treasury expansion (`tau`)
+* minimum fixed rewards cut for pools (`minPoolCost`)
+* minimum Lovelace deposit per byte of serialized UTxO (`coinsPerUTxOByte`)
+* prices of Plutus execution units (`prices`)
+
+The **technical group** consists of:
+* pool pledge influence (`a0`)
```

Comment:
Isn't this (and `nOpt`/k) more an economic parameter?

The distribution of parameters between economic and technical seems rather arbitrary.

Maybe it would make sense to rather use the topic of the parameters as groups, all parameters for treasury and reserve under “monetary policy”, all parameters for staking and rewards under “stake pool”, all parameters for basic transactions under “transaction”, all parameters for Plutus scripts under “Plutus”, …